docs/documentation/concepts/discovery.md

```diff
@@ -18,16 +18,16 @@ It is possible to override this setting and deactivate background discovery for
 
 ## Inbox
 
-The inbox holds a list of all discovered things (`DiscoveryResult`) from all active discovery services. A discovery result represents a discovered thing of a specific thing type, that could be instantiated as a thing. The result usually contains properties that identify the discovered things further like IP address or a serial number. Each discovery result also has a timestamp when it was added to or updated in the inbox and it may also contain a time to live, indicating the time after which it is be automatically removed from the inbox. 
+The inbox holds a list of all discovered Things (`DiscoveryResult`) from all active discovery services. A discovery result represents a discovered Thing of a specific Thing type, that could be instantiated as a Thing. The result usually contains properties that identify the discovered Things further like IP address or a serial number. Each discovery result also has a timestamp when it was added to or updated in the inbox and it may also contain a time to live, indicating the time after which it is be automatically removed from the inbox. 
```

Comment:
Could you please break up this paragraph into multiple lines, i.e. add a line break after each sentence?